python/docs/conf.py

```diff
@@ -1,6 +1,4 @@
 #!/usr/bin/env python
-# -*- coding: utf-8 -*-
-# flake8: noqa
 #
 # pynessie documentation build configuration file, created by
```

Comment:
this file seems to have no license header at all?
do we know why the check is not enforced here?

Reply:
pylint checks are being run on pynessie, tools, and tests folders. This file is in docs and not checked.

Reply:
hmm thats odd, because `# flake8: noqa` means that flake8 supposedly runs against this file... so if it does probably also pylint should

Reply:
The folders are mentioned explicitly in the pylint command, should I change it to run on docs as well?

Reply:
hmm no its fine then, but then, since we do not run flake8 on the docs folder, we can just remove the `# flake8: noqa` (probably a leftover from a time when docs folder was still checked)